packages/desktop/src/main/index.ts

```diff
@@ -82,6 +82,29 @@ const createMainWindow = () => {
         mainWindow = undefined;
     });
 
+    // Customize window.open or target="_blank" calls that create new Electron windows
```

Comment:
maybe add a labkey specifc comment so that if something changes its easier for devs to pick up?

Reply:
added something in [ed545f2](https://github.com/AllenInstitute/biofile-finder/pull/514/commits/ed545f22479263b3218dc9c8142af30d20da090e)